templates/cisco_nxos_show_ip_interface.textfsm

```diff
@@ -0,0 +1,105 @@
+Value Filldown VRF_NAME (\S+)
+Value Required INTERFACE (\S+)
+Value PROTOCOL_STATUS (\w+)
+Value LINK_STATUS (\w+)
+Value ADMIN_STATUS (\w+)
+Value PRIMARY_IP_ADDRESS (\S+)
+Value PRIMARY_IP_SUBNET (\S+)
+Value PRIMARY_BROADCAST_ADDRESS (\S+)
+Value List SECONDARY_IP_ADDRESS (\S+)
+Value List SECONDARY_IP_SUBNET (\S+)
+Value MULTICAST_GROUPS (224\..*?)
+Value MTU (\d+)
+Value ROUTE_PREFERENCE (\d+)
+Value ROUTE_PREFERENCE_TAG (\d+)
+Value PROXY_ARP (\S+)
+Value LOCAL_PROXY_ARP (\S+)
+Value MULTICAST_ROUTING (\S+)
+Value ICMP_REDIRECTS (\S+)
+Value DIRECTED_BROADCAST (\S+)
+Value IP_FORWARDING (\S+)
+Value ICMP_UNREACHABLES (\S+)
+Value ICMP_PORT_UNREACHABLE (\S+)
+Value URPF (\S+)
+Value LOAD_SHARING (\S+)
+Value LAST_RESET (\S+)
+Value UNICAST_PACKETS_SENT (\d+)
+Value UNICAST_PACKETS_RECEIVED (\d+)
+Value UNICAST_PACKETS_FORWARDED (\d+)
+Value UNICAST_PACKETS_ORIGINATED (\d+)
+Value UNICAST_PACKETS_CONSUMED (\d+)
+Value UNICAST_BYTES_SENT (\d+)
+Value UNICAST_BYTES_RECEIVED (\d+)
+Value UNICAST_BYTES_FORWARDED (\d+)
+Value UNICAST_BYTES_ORIGINATED (\d+)
+Value UNICAST_BYTES_CONSUMED (\d+)
+Value MULTICAST_PACKETS_SENT (\d+)
+Value MULTICAST_PACKETS_RECEIVED (\d+)
+Value MULTICAST_PACKETS_FORWARDED (\d+)
+Value MULTICAST_PACKETS_ORIGINATED (\d+)
+Value MULTICAST_PACKETS_CONSUMED (\d+)
+Value MULTICAST_BYTES_SENT (\d+)
+Value MULTICAST_BYTES_RECEIVED (\d+)
+Value MULTICAST_BYTES_FORWARDED (\d+)
+Value MULTICAST_BYTES_ORIGINATED (\d+)
+Value MULTICAST_BYTES_CONSUMED (\d+)
+Value BROADCAST_PACKETS_SENT (\d+)
+Value BROADCAST_PACKETS_RECEIVED (\d+)
+Value BROADCAST_PACKETS_FORWARDED (\d+)
+Value BROADCAST_PACKETS_ORIGINATED (\d+)
+Value BROADCAST_PACKETS_CONSUMED (\d+)
+Value BROADCAST_BYTES_SENT (\d+)
+Value BROADCAST_BYTES_RECEIVED (\d+)
+Value BROADCAST_BYTES_FORWARDED (\d+)
+Value BROADCAST_BYTES_ORIGINATED (\d+)
+Value BROADCAST_BYTES_CONSUMED (\d+)
+Value LABELED_PACKETS_SENT (\d+)
+Value LABELED_PACKETS_RECEIVED (\d+)
+Value LABELED_PACKETS_FORWARDED (\d+)
+Value LABELED_PACKETS_ORIGINATED (\d+)
+Value LABELED_PACKETS_CONSUMED (\d+)
+Value LABELED_BYTES_SENT (\d+)
+Value LABELED_BYTES_RECEIVED (\d+)
+Value LABELED_BYTES_FORWARDED (\d+)
+Value LABELED_BYTES_ORIGINATED (\d+)
+Value LABELED_BYTES_CONSUMED (\d+)
+Value WCCP_REDIRECT_OUTBOUND (\S+)
+Value WCCP_REDIRECT_INBOUND (\S+)
+Value WCCP_REDIRECT_EXCLUDE (\S+)
+
+Start
+  ^IP\s+Interface\s+Status\s+for\s+VRF\s+\"${VRF_NAME}\"\s*$$
```

Comment:
```suggestion
Start
  ^IP\s+Interface\s+Status\s+for\s+VRF\s+\"${VRF_NAME}\"\s*$$
  ^\S+, -> Continue.Record
```

Remove the record on WCCP line.

Reply:
Wouldn't this also work ?

^${INTERFACE},.*protocol-${PROTOCOL_STATUS}\/link-${LINK_STATUS}\/admin-${ADMIN_STATUS},.*$$ -> Continue.Record

Reply:
Unfortunately this doesn't work, some interfaces get the wrong VRF value